### PR TITLE
Update agile-opensource-workflow.md

### PR DESCRIPTION
**Pull request by EllenDevereux**
I've made some updates, let me know what you think

### CODE REVIEW

File: project-management/agile-opensource-workflow.md [hunk 1]
```diff
@@ -29,30 +33,34 @@ The tasks and user stories are prioritised (with the input of members of the aut
 
 The issues that correspond to the backlog are transferred to the Fujitsu team's fork of the GitHub repository.
 
+### 4. Completion of tasks
+
 Completed tasks are submitted as pull requests to the main repository.
 
 The remaining user stories remain open as issues on GitHub for anyone to develop or contribute to on an ad hoc continual basis.
```
Comment (GeorgiaHCA) [inline project-management/agile-opensource-workflow.md]: continuos

Comment (KirstieJane) [reply project-management/agile-opensource-workflow.md]:
```suggestion
The remaining user stories remain open as issues on GitHub for anyone to develop or contribute to on an ad hoc continuous basis.
```

File: project-management/agile-opensource-workflow.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ This document describes a workflow that combines participatory, open source and
 *There needs to be something here that provides a little background about why we need a whole new way of working.*
 *Something along the lines of the difference between continual and continuous, and the difference between people who are paid to work on the project vs people who are volunteering their time - both members of the autistic community and volunteer developers.*
 
+This is a project funded by and supported by the Autistic community and therefore every stage of the project requires complete transparency and community feedback. A new way of working needs to be developed to incorporate Alan Turing and Fujitsu to work effectively together. Fujitsu work in an Agile way to develop the platform efficiently.
```
Comment (KirstieJane) [inline project-management/agile-opensource-workflow.md]:
```suggestion
This is a project funded by and supported by the Autistic community and therefore every stage of the project requires complete transparency and community feedback.
A new way of working needs to be developed to incorporate Alan Turing and Fujitsu to work effectively together.
All parties are committed to working in an Agile (and agile) way to develop the platform efficiently.
```

Comment (KirstieJane) [reply project-management/agile-opensource-workflow.md]: incorporating @JamesSCTJ's suggestion (and new line formatting).

File: project-management/agile-opensource-workflow.md (hunk 1 above)
Comment (KirstieJane) [reply project-management/agile-opensource-workflow.md]: I'll have to commit this separately as I don't have permission to commit to @EllenDevereux's fork.

File: project-management/agile-opensource-workflow.md (hunk 2 above)
Comment (KirstieJane) [reply project-management/agile-opensource-workflow.md]: I'll have to commit this separately as I don't have permission to commit to @EllenDevereux's fork.

Comment (KirstieJane) [reply project-management/agile-opensource-workflow.md]: Done!

File: project-management/agile-opensource-workflow.md (hunk 1 above)
Comment (KirstieJane) [reply project-management/agile-opensource-workflow.md]: Done

**Pull request merged**